BattleSim/src/main/java/com/revature/repositories/RolesRepository.java

```diff
@@ -0,0 +1,14 @@
+package com.revature.repositories;
+
+import java.util.List;
+
+import javax.management.relation.Role;
+
+public interface RolesRepository {
+	public int addRole(Role r);
+	public List<Role> getAllRoles();
+	public Role getRole(int id);
+//	public Role getRole(name);
+	public void updateRoles(Role change);
+	public void deleteRoles(int id);
```

Comment:
Plural unnecessary

Reply:
It looks like this plural from updateRoles and deleteRoles is still here. Suggest updateRole and deleteRole.